src/main/java/io/temporal/internal/grpc/GrpcTracingInterceptor.java

```diff
@@ -49,7 +49,7 @@
         next.newCall(method, callOptions)) {
       @Override
       public void sendMessage(ReqT message) {
-        log.trace("Invoking " + method.getFullMethodName() + "with input: " + message);
+        log.trace("Invoking \"" + method.getFullMethodName() + "\" with input: " + message);
```

Comment:
[Nit] dumb question (based on similar change above), when do we use single quote (') vs double quote (") for these types of messages?

Reply:
There is no guideline here. I would support coming up with one and making sure that all the log messages obey it.